Makefile

```diff
@@ -49,12 +49,19 @@ endif
 VENV := "venv/bin/activate"
 
 .PHONY: site
-site: bibtex mods endnote hugo split-sitemap test
+site: bibtex mods endnote hugo sitemap
 
 
-.PHONY: split-sitemap
-split-sitemap: venv/bin/activate
+# Split the file sitemap into Google-ingestible chunks.
+# Also build the PDF sitemap, and split it.
+.PHONY: sitemap
+sitemap: build/.sitemap
+
+build/.sitemap: venv/bin/activate $(sourcefiles)
 	. $(VENV) && python3 bin/split_sitemap.py build/anthology/sitemap.xml
+	@gzip -9n build/anthology/sitemap_*.xml
+	@bin/create_sitemapindex.sh `ls build/anthology/sitemap_*.xml.gz` > build/anthology/sitemapindex.xml
```

Comment:
Won't this create URLs like `https://www.aclweb.org/anthology/build/anthology/sitemap_1.xml.gz` in the `sitemapindex.xml`?

Reply:
You are right.  I must have commited a wrong version (see also the "echo gzip" I commited ...).  Should be `ls build/anthology/ | grep 'sitemap_.*xml.gz'`.  Fixed.